libvast/src/system/explorer.cpp

```diff
@@ -168,14 +177,14 @@ explorer(caf::stateful_actor<explorer_state>* self, node_actor node,
           continue;
         std::optional<vast::expression> before_expr;
         if (st.before)
-          before_expr = predicate{attribute_extractor{atom::timestamp_v},
+          before_expr = predicate{type_extractor{none_type{}.name("timestamp")},
                                   relational_operator::greater_equal,
                                   data{*x - *st.before}};
 
         std::optional<vast::expression> after_expr;
         if (st.after)
           after_expr
-            = predicate{attribute_extractor{atom::timestamp_v},
+            = predicate{type_extractor{none_type{}.name("timestamp")},
```

Comment:
Why `none_type` and not `time_type`?

Reply:
The `type_resolver` should handle both cases correctly. I'll change it to `time_type`.